docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.8"
+version: '3.8'
```

Comment:
nit: are we accidentally running a formatter on this? 

Either quote style is fine, but it's better to not mix in arbitrary style changes.

Reply:
(Since we're not using prettier in this repo, it may be a good idea to _turn off_ Prettier in VS Code for this workspace)

Reply:
I have Prettier turned off and ran `yarn lint --fix`. It still uses single quotes. 🤷🏾‍♀️

Reply:
Eslint does not run on .yml files, I am guessing an editor save did it.